README.md

```diff
@@ -136,38 +136,39 @@ then change it to
         }
     
         setSpeed(allowedSpeed);
-    
+```
```

Comment:
AFAIK, Code snippet is shown on two cases, either three back quotes or spaces. and it has both, which resulted in next text to be shown as code snipped i.e "Simplified as". it is happening at multiple places, could you please fix it.

[Not because of your change] What do you think, about codes not having initial tabs in code snippets? right now, it is at all the places.